src/main/java/utils/driver/IOSDriverManager.java

```diff
@@ -1,15 +1,15 @@
 package utils.driver;
 
 import io.appium.java_client.AppiumDriver;
-import io.appium.java_client.android.AndroidDriver;
+import io.appium.java_client.ios.IOSDriver;
 import io.appium.java_client.service.local.AppiumDriverLocalService;
 
 import java.util.concurrent.TimeUnit;
 
-public class AndroidDriverManager extends DriverManager {
+public class IOSDriverManager extends DriverManager {
     @Override
     public AppiumDriver createDriver(AppiumDriverLocalService service) {
-        driver = new AndroidDriver(service, cap);
+        driver = new IOSDriver(service, cap);
```

Comment:
if the app is both for android and iOS it will be better to use AppiumDriver instead of iOS/Android driver here